packages/neuron-wallet/src/services/cells.ts

```diff
@@ -325,6 +336,41 @@ export default class CellsService {
       .orderBy('tx.timestamp', 'ASC')
       .getMany()
 
+    // datum in outputs has been spliced when sync
```

Comment:
Better to append the link to the code so it would be easy to track this annotation

Reply:
I've appended a comment for the reason RPC is required here